src/lib/datepicker/calendar.ts

```diff
@@ -63,19 +63,31 @@ export class MdCalendar<D> implements AfterContentInit, OnDestroy {
   private _intlChanges: Subscription;
 
   /** A date representing the period (month or year) to start the calendar in. */
-  @Input() startAt: D;
+  @Input()
+  get startAt(): D { return this._startAt; }
+  set startAt(value: D) { this._startAt = this._coerceDateProperty(value); }
```

Comment:
Is it weird that the user could do
```ts
picker.startAt = '1985-04-12T23:20:50.52Z';
console.log(picker.startAt); // is a different type
```
?

Reply:
Kind of? but its the same idea as all of the boolean and number properties that we coerce